src/python/shorah_dec.py

```diff
@@ -25,6 +25,11 @@
     file of corrected reads
 '''
 
+from __future__ import print_function
+from __future__ import division
+from builtins import dict
+from future.utils import listitems
+from builtins import range
```

Comment:
@ozagordi from the list http://python-future.org/compatible_idioms.html#dict-keys-values-items-as-a-list I don't see an idiomatic way to handle both Py2+3 APIs? maybe just do `itemlist = list(heights.items())` and bite the performance bullet for Py2?